### PR TITLE
Removed destination id from get thirdparty transaction.

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: src/lib/requests/thirdpartyRequests.js [hunk 1]
```diff
@@ -12,9 +12,9 @@ class ThirdpartyRequests extends BaseRequests {
         return this._post('authorizations', 'authorizations', authorizationBody, destParticipantId);
     }
 
-    async getThirdpartyRequestsTransactions(transactionRequestId, destParticipantId) {
+    async getThirdpartyRequestsTransactions(transactionRequestId) {
         const url = `thirdpartyRequests/transactions/${transactionRequestId}`;
-        return this._get(url, 'thirdparty', destParticipantId);
+        return this._get(url, 'thirdparty');
```
Comment (eoln) [inline src/lib/requests/thirdpartyRequests.js:17]: @kleyow & @lewisdaly do we really want to drop fspiop-destination header? How will we handle the case when multiple pisp will be onboarded?

**Comment by lewisdaly** [comment]
Hmm, I'm not sure I understand this one @kleyow

**Comment by kleyow** [comment]
@eoln @lewisdaly Isn't typically the micro-services responsibility to find out who has a specific resource? 

Like how the als handles GET `/parties` calls or ml-api-adapter handles GET `/transfers` requests. When a participant wants to look up a third-party transaction they don't know who has that information right?

When fspiop-destination is null the request is just sent directly `config.thirdpartyRequestsEndpoint` yes?

**Comment by kleyow** [comment]
That's the impression I got from seeing other `GET` calls in https://github.com/mojaloop/sdk-standard-components/blob/master/src/lib/requests/mojaloopRequests.js

**Comment by lewisdaly** [comment]
> That's the impression I got from seeing other `GET` calls in https://github.com/mojaloop/sdk-standard-components/blob/master/src/lib/requests/mojaloopRequests.js

Ah I see. I think I understand what is going on here. There's no sequence diagram in the API Definition for `GET /transfers`, but I think that the switch is the "source of truth for a transfer in this case.

The case that I think we want is where the PISP/DFSP is able to have their `GET thirdpartyRequests/transactions/${transactionRequestId}` routed to the right place, more like `GET /transactions/{id}` [see the spec here](https://docs.mojaloop.io/mojaloop-specification/documents/API%20Definition%20v1.0.html#68-api-resource-transactions).

**Comment by kleyow** [comment]
@lewisdaly Yea the switch should handle the routing. The sender PISP/DFSP doesn't know who has the corresponding information so the destination header isn't need. (I believe that to be the case but I'm still a bit grey on the inner workings of the switch).

Or are you disagreeing with me? I can't quite pick out the conclusion of your inner monologue.

**Comment by lewisdaly** [comment]
I was disagreeing with you, but now I'm not so sure. 

The challenge here is that the thirdparty-api-adapter has _no_ state. So it can't know who to route the `GET` to. So that's why I was thinking we _need_ the `destParticipantId`. But I do take your point about how the ideal scenario for a DFSP/PISP would be to just send a `GET` without the `destParticipantId`, and let the switch worry about it.

In all honesty, I don't think this is too critical at this point - perhaps we can table it for now and open up a new ticket to clarify this.

**Comment by kleyow** [comment]
@lewisdaly Sounds good!

It might result is some questions when I handle GET thirdpartyRequest/transaction/{ID} in https://github.com/mojaloop/mojaloop/issues/314 but I agree we can shelve this for now.

**Pull request closed without merging**